### PR TITLE
FIX#88411 tablature tie position

### PR DESCRIPTION
**Pull request by sidewayss**
Replaces PR #4673 which closed itself somehow.
Fixes this issue: https://musescore.org/en/node/88411

### CODE REVIEW

File: libmscore/tie.cpp [hunk 1]
```diff
@@ -287,8 +287,11 @@ void TieSegment::computeBezier(QPointF p6o)
       shapePath.cubicTo(p4 +p4o + th, p3 + p3o + th, QPointF());
 
       // translate back
+      double y = pp1.y();
+      if (staff()->isTabStaff(slurTie()->tick()))
+          y += (_spatium * (slurTie()->up() ? -0.2 : 0.2));
```
Comment (anatoly-os) [inline libmscore/tie.cpp]: Could you please add the variable which describes the meaning of `0.2` magic number?

Comment (sidewayss) [reply libmscore/tie.cpp]: OK.  It's not very magical. It's my best guess.  I asked for feedback and got none, so here it is. I'll declare it as a const double.

Comment (sidewayss) [reply libmscore/tie.cpp]: I named it `offsetFactor`.  This is an extra offset for tablature.

Comment (sidewayss) [reply libmscore/tie.cpp]: See here for a similar, existing line of code:
https://github.com/musescore/MuseScore/blob/b3adb50bd655bc943c502af8d6dc22fd9422f81a/libmscore/note.cpp#L1116
That's where I got the idea for this offset.

**Pull request merged**